src/utils/gql/GQLWrapper.tsx

```diff
@@ -58,13 +60,26 @@ export const getClientLink = async ({
 
 const cache = new InMemoryCache();
 
+const authLink = (logout: Logout) =>
+  onError(({ networkError }) => {
```

Comment:
I'm unsure the reasoning behind logging the user out if there is a 401

Reply:
because 401 stands for unauthorized